openstack/manila/alerts/openstack-manila.alerts

```diff
@@ -71,6 +71,33 @@ groups:
       description: Manila Shares taking more than 2 minutes to resize in {{ $labels.manila_host }}
       summary: Manila Shares taking more than 2 minutes to resize
 
+  - alert: OpenstackManilaShareServersStuckCreate
+    expr: max(openstack_manila_share_servers_stuck_max_duration_gauge{status="creating"})
+      BY (manila_host) > 120
+    for: 5m
+    labels:
+      dashboard: manila
+      service: manila
+      severity: info
+      tier: os
+    annotations:
+      description: Manila Share server {{ $labels.id }} taking more than 2 minutes to create on {{ $labels.manila_host }}
+      summary: Manila Share servers taking more than 2 minutes to create
+
+  - alert: OpenstackManilaShareServersStuckError
+    expr: max(openstack_manila_share_servers_stuck_max_duration_gauge{status=~"creating|deleting"})
```

Comment:
I think the idea was to have second alert for Error/Error_Manage/etc states?
Because alert is called: OpenstackManilaShareServersStuckError

Reply:
yes, separate for Error* and Creating.

Reply:
in such case status=~"creating|deleting" has to be changed: status=~"error|error_unmanage|error_manage", right?